File: src/main/kotlin/com/github/tobi812/sprykerplugin/models/finder/ClassFinderInterface.java
package com.github.tobi812.sprykerplugin.models.finder;

import pav.sprykerFileCreator.model.renderer.dto.PhpClassInterface;

public interface ClassFinderInterface {

    Boolean doesClassExist(String fullQualifiedName);

    PhpClassInterface findClass(String fullQualifiedCoreName);

}

File: src/main/kotlin/com/github/tobi812/sprykerplugin/models/finder/ClassFinderInterface.kt
package com.github.tobi812.sprykerplugin.models.finder

import com.github.tobi812.sprykerplugin.models.renderer.dto.PhpClassInterface
import com.jetbrains.php.lang.psi.elements.PhpClass

interface ClassFinderInterface {
    fun doesClassExist(fullQualifiedName: String): Boolean
    fun findClass(fullQualifiedCoreName: String): PhpClassInterface?
    fun findPhpClassCollection(fullQualifiedName: String): Collection<PhpClass?>
}
